tests/core/test_snapshot.py

```diff
@@ -98,7 +99,7 @@ def test_json(snapshot: Snapshot):
             "name": "name",
             "partitioned_by": [],
             "owner": "owner",
-            "query": "SELECT @EACH(ARRAY(1, 2), x -> x), ds FROM parent.tbl",
+            "query": "SELECT @EACH([1, 2], x -> x), ds FROM parent.tbl",
```

Comment:
This test needed to be changed because we're not actually calling `parse_raw` to get back the spark version of the array AST node, but rather just doing a simple json loads which will give us back whatever was stored in the `"sql"` entry of the `meta` map (in this case, it was `@EACH([1, 2], ....)`.